diff --git a/runtime/CMakeLists.txt b/runtime/CMakeLists.txt
index 86fdab9..e512fcd 100644
--- a/runtime/CMakeLists.txt
+++ b/runtime/CMakeLists.txt
@@ -62,7 +62,7 @@ if (ANTLR_BUILD_CPP_TESTS)
 
   FetchContent_Declare(
     googletest
-    URL https://github.com/google/googletest/archive/e2239ee6043f73722e7aa812a459f54a28552929.zip
+    SOURCE_DIR @gtest_src@
   )
 
   if(WITH_STATIC_CRT)
@@ -158,7 +158,7 @@ if (ANTLR_BUILD_CPP_TESTS)
       COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_FILE:antlr4_shared> ${CMAKE_HOME_DIRECTORY}/dist
       COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_LINKER_FILE:antlr4_shared> ${CMAKE_HOME_DIRECTORY}/dist)
   endif()
-  
+
   if (TARGET antlr4_static)
     add_custom_command(
       TARGET antlr4_static
