diff --git a/fasm/parser/__init__.py b/fasm/parser/__init__.py
index edd02e4..9d188e0 100644
--- a/fasm/parser/__init__.py
+++ b/fasm/parser/__init__.py
@@ -22,25 +22,8 @@ from warnings import warn
 available = []
 """ List of parser submodules available. Strings should match module names. """
 
-try:
-    from fasm.parser.antlr import \
-        parse_fasm_filename, parse_fasm_string, implementation
-    available.append('antlr')
-except ImportError as e:
-    warn(
-        """Unable to import fast Antlr4 parser implementation.
-  ImportError: {}
-
-  Falling back to the much slower pure Python textX based parser
-  implementation.
-
-  Getting the faster antlr parser can normally be done by installing the
-  required dependencies and then reinstalling the fasm package with:
-    pip uninstall
-    pip install -v fasm
-""".format(e), RuntimeWarning)
-    from fasm.parser.textx import \
-        parse_fasm_filename, parse_fasm_string, implementation
+from fasm.parser.textx import \
+    parse_fasm_filename, parse_fasm_string, implementation
 
 # The textx parser is available as a fallback.
 available.append('textx')
